eZ/Publish/Core/Persistence/Cache/Tests/ContentHandlerTest.php

```diff
@@ -46,12 +46,12 @@ public function providerForUnCachedMethods(): array
             ['createDraftFromVersion', [2, 1, 14], ['content-2-version-list']],
             ['copy', [2, 1]],
             ['loadDraftsForUser', [14]],
-            ['setStatus', [2, 0, 1], ['content-2-version-list'], 'ez-content-version-info-2-1'],
+            ['setStatus', [2, 0, 1], ['content-2-version-list', 'content-2-version-1'], 'ez-content-version-info-2-1'],
```

Comment:
Review side note on this one:
We can probably consider removing invalidation of `content-X-version-list` here, as the list should contain the new `'content-X-version-Y` now anyway. Also can consider skipping call to `$this->cache->deleteItem("ez-content-version-info-${contentId}-${versionNo}");`, it should be covered by tag clearing now.